Cannot redeclare block-scoped variable 'Symbol'

If using `@types/node` and this package we have problems since https://github.com/benlesh/symbol-observable/pull/29:

> node_modules/@types/node/index.d.ts(67,13): error TS2451: Cannot redeclare block-scoped variable 'Symbol'.
node_modules/symbol-observable/index.d.ts(9,16): error TS2451: Cannot redeclare block-scoped variable 'Symbol'.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/222185d8afbcf4dbc727e276af8aeb7ef3543321/types/node/index.d.ts#L80 and https://github.com/benlesh/symbol-observable/blob/a17f22878b0cfcdb2ab161de31a1569f37d417c7/index.d.ts#L9 have issues 😢

--- a/index.d.ts
+++ b/index.d.ts
@@ -5,8 +5,6 @@ declare global {
   export interface SymbolConstructor {
     readonly observable: symbol;
   }
-
-  export const Symbol: SymbolConstructor;
 }
 
 export interface Symbol {

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "npm run build && mocha && tsc ./ts-test/test.ts && node ./ts-test/test.js && check-es3-syntax -p lib/ --kill",
+    "test": "npm run build && mocha && tsc && node ./ts-test/test.js && check-es3-syntax -p lib/ --kill",
     "build": "babel es --out-dir lib",
     "prepublish": "npm test"
   },

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,0 +1,16 @@
+{
+  "include": [
+    "./ts-test"
+  ],
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "strict": true,
+    "lib": [
+      "dom",
+      "es5",
+      "scripthost",
+      "es2015.symbol"
+    ]
+  }
+}

